examples/06_deduplication.py

```diff
@@ -90,6 +85,12 @@
 # a lighter color means a larger distance. We can see that we are dealing with three
 # clusters - the original medication names and their misspellings that cluster around them.
 
+from dirty_cat._deduplicate import compute_ngram_distance
```

Comment:
```suggestion
from dirty_cat import compute_ngram_distance
```
This function was actually made public and added to the init, so there is no need to import it from a private submodule #466